packages/wonder-stuff-core/src/__tests__/kind-error.test.js

```diff
@@ -240,6 +273,30 @@ describe("KindError", () => {
                 // Assert
                 expect(error.message).toBe(combinedErrorInfo.message);
             });
+
+            it("should have a combined message that cascades all causes", () => {
+                // Arrange
+                const cause1 = new Error("CAUSE_MESSAGE");
+                const cause2 = new KindError(
+                    "CAUSE_2_MESSAGE",
+                    Errors.Unknown,
+                    {cause: cause1},
+                );
+
+                // Act
+                const result = new KindError("ROOT_MESSAGE", Errors.Unknown, {
```

Comment:
I feel like `ROOT` implies this is at the bottom of the nesting, not the top.

Reply:
Hmm, I was thinking of it like with directory structures. 🤷🏻